src/uct/ib/ud/base/ud_ep.c

```diff
@@ -1401,7 +1401,7 @@ static void uct_ud_ep_send_ack(uct_ud_iface_t *iface, uct_ud_ep_t *ep)
         goto out;
     }
 
-    if (uct_ud_ep_ctl_op_check(ep, UCT_UD_EP_OP_SIGNALED)) {
+    if (uct_ud_ep_ctl_op_check(ep, UCT_UD_EP_OP_ACK_REQ)) {
```

Comment:
should be UCT_UD_EP_OP_FLUSH?

Reply:
maybe it worth sending `UCT_UD_EP_OP_ACK_REQ` as signaled always as I proposed in this commit?
changing it to FLUSH leads to updating many places to check not only ACK_REQ, but also FLUSH.

Reply:
seems we don't set UCT_UD_EP_OP_FLUSH at all anywhere..
so can remove UCT_UD_EP_OP_FLUSH altogether 
i guess ACK with ACK_REQ case is rare enough (only with timeout or flush) so can ask signal for any ACK wit hACK_REQ

Reply:
sorry, I forgoto to add the code which removes FLUSH to the commit